cleansweep/volunteers/views.py

```diff
@@ -33,6 +34,19 @@ def add_volunteer(place):
     return render_template("add_volunteer.html", place=place, form=form)
 
 
+@plugin.place_view("/volunteers.xls", permission="view-volunteers")
+def download_volunteer(place):
+    headers = ['Name', "Phone", 'Email', 'Voter ID', 'Location']
+    data = tablib.Dataset(headers=headers)
+    for m in place.get_all_members():
+        data.append([m.name, m.phone, m.email, m.voterid, m.place.name])
+    response = make_response(data.xls)
+    response.headers['content_type'] = 'application/vnd.ms-excel;charset=utf-8'
+    response.headers['Content-Disposition'] = "attachment; filename='{0}-volunteers.xls'".format(place.key)
```

Comment:
@anandology  I also thought that, but that seems to working as alright and generate the required format.